G7SensorKitUI/Views/G7SettingsView.swift

```diff
@@ -173,8 +175,15 @@ struct G7SettingsView: View {
                     .foregroundColor(color(for: viewModel.progressBarState.labelColor))
 
                 Spacer()
+                var sessionLengthFormatter: DateComponentsFormatter = {
```

Comment:
View composition shouldn't have significant blocks of non-view code; it makes it harder to see what is layout and what is logic.  And the old durationFormatter is still left in the code, unused.  Probably better to move the contents of this formatter block up to where durationFormatter was defined.

Reply:
Fixed this. Moved sessionLengthFormatter out of view. Removed durationFormatter.